util/feeding.py

```diff
@@ -29,28 +29,31 @@ def read_csvs(csv_files):
     return source_data
 
 
-def samples_to_mfccs(samples, sample_rate):
+def samples_to_mfccs(samples, sample_rate, spec_augment=False):
     spectrogram = contrib_audio.audio_spectrogram(samples,
                                                   window_size=Config.audio_window_samples,
                                                   stride=Config.audio_step_samples,
                                                   magnitude_squared=True)
-    mfccs = contrib_audio.mfcc(spectrogram, sample_rate, dct_coefficient_count=Config.n_input)
-    mfccs = tf.reshape(mfccs, [-1, Config.n_input])
+    if spec_augment:
```

Comment:
The parameter shadows the function, so this ends up causing a `TypeError: 'bool' object is not callable`